content/applications/hr/payroll.rst

```diff
@@ -257,10 +267,9 @@ like expenses, reimbursements, or deductions. These other inputs can be configur
    :align: center
    :alt: Other input types for payroll.
 
-To create a new input type, click the :guilabel:`Create` button. Enter the :guilabel:`Description`,
-the :guilabel:`Code`, and which structure it applies to in the :guilabel:`Availability in Structure`
-field. Click the :guilabel:`Save` button to save the changes, or click :guilabel:`Discard` to delete
-the entry.
+To create a new input type, click the :guilabel:`New` button. Enter the :guilabel:`Description`, the
+:guilabel:`Code`, and which structure it applies to in the :guilabel:`Availability in Structure`
+field.
```

Comment:
What does entering these fields mean? I can see that we need to do this. The documentation should have extra info for the user.

Reply:
As a layman to Payroll this doc is somewhat confusing to me. Pretend you're writing to someone who is trying to configure Odoo on their own.